test/Main.purs

```diff
@@ -20,9 +20,9 @@ import Test.Spec.Runner (runSpec)
 foreign import _configureJsDomViaFfi :: Type
 
 main :: Effect Unit
-main = do
+main = launchAff_ $ do
   Enzyme.configure Adapter.unofficialReact_17
```

Comment:
Now that `unofficialReact_17 :: Aff Adapter`, I don't think this line would compile.

Reply:
i was so focussed on getting it running in elmish that I forgot to update this test 😅. fixed now